Project/Galagram/Converters/ReverseBooleanConverter.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Windows.Data;
+
+namespace Galagram.Converters
+{
+    /// <summary>
+    /// Converts bool to its opposite value
+    /// </summary>
+    [ValueConversion(sourceType: typeof(object[]), targetType: typeof(object))]
+    public class ReverseBooleanConverter : IValueConverter
```

Comment:
This one will be deleted later 😞  forgot about it

Reply:
Okay